tox.ini

```diff
@@ -17,7 +17,10 @@ commands =
     pre-commit run flake8 --all-files --show-diff-on-failure
     pre-commit run isort --all-files --show-diff-on-failure
     pre-commit run black --all-files --show-diff-on-failure
-    # mypy . #issue33
+
+[testenv:typing]
+commands =
+    mypy .
```

Comment:
Why not run `mypy` on each pr if we are going to introduce linting?
@kairoaraujo do you agree?
If @KAUTH has spent so much time on this (and I personally strongly support this change) why not move to `mypy` linting for each of our changes as well?

Reply:
Yes, I agree. 🎉 

add `typing` to 
https://github.com/vmware/repository-service-tuf-cli/blob/3e500ea55b00272d197b1330fd75ca3262eefc22/tox.ini#L3
and
https://github.com/vmware/repository-service-tuf-cli/blob/3e500ea55b00272d197b1330fd75ca3262eefc22/tox.ini#L48-L51

@KAUTH integrate also to pre-commit 🚀

Reply:
Since the official `mypy` repo doesn't provide a `pre-commit` hook we have to resort to [`mirrors-mypy`](https://github.com/pre-commit/mirrors-mypy) which runs `mypy` from an isolated virtualenv and is not as flexible as `mypy` (e.g., cannot exclude specific modules as we do in the `pyproject.toml` mypy config).
For this reason I am creating a local hook to run `mypy` from.